pandas/tests/io/json/test_compression.py

```diff
@@ -33,24 +33,19 @@ def test_read_zipped_json(datapath):
 
 
 @td.skip_if_not_us_locale
-def test_with_s3_url(compression):
-    boto3 = pytest.importorskip('boto3')
-    pytest.importorskip('s3fs')
-    moto = pytest.importorskip('moto')
+def test_with_s3_url(compression, s3_resource):
+    # Bucket "pandas-test" created in tests/io/conftest.py
 
     df = pd.read_json('{"a": [1, 2, 3], "b": [4, 5, 6]}')
-    with moto.mock_s3():
-        conn = boto3.resource("s3", region_name="us-east-1")
-        bucket = conn.create_bucket(Bucket="pandas-test")
 
-        with tm.ensure_clean() as path:
```

Comment:
this was mocked before, why are you now non-mocking it? this causes permission issues.

Reply:
No, it's still mocked, because the function now uses the `s3_resource` fixture, which does the mocking.

So now, all s3 tests use the fixture instead of doing their own mocking.